raidpackage.py

```diff
@@ -0,0 +1,155 @@
+    return sent_message, preorder_embed
+
+async def wait_for_order_reaction_add(payload, client, sent_message, preorder_embed, usr, order_channel):  
+
+    if str(payload.emoji) == icon_definitions.emoji_spectral_str:
+        item = "Potion of Spectral Strength"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 40, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value=item + f" x{qtyReq}", inline=False))
+            
+    if str(payload.emoji) == icon_definitions.emoji_spectral_int:
+        item = "Potion of Spectral Intellect"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 40, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value=item + f" x{qtyReq}", inline=False))
+
+    if str(payload.emoji) == icon_definitions.emoji_shaded_sharpen:
+        item = "Shaded Sharpening Stone"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 6, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value=item + f" x{qtyReq}", inline=False))
+
+    if str(payload.emoji) == icon_definitions.emoji_shaded_weightstone:
+        item = "Shaded Weightstone"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 6, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value=item + f" x{qtyReq}", inline=False))
+
+async def wait_for_order_reaction_remove(reaction_remove_payload, sent_message, preorder_embed):
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_spectral_str:
+        await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_spectral_int:
+        await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_shaded_sharpen:
+        await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_shaded_weightstone:
+        await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value='None', inline=False))  
+
+async def cancel_order(msg, usr_id, client):
+    await msg.delete()
+    cancelMsg = discord.Embed(title="Order Cancelled", url='', color=0x109319)
```

Comment:
Rather than creating a new message, it would be more efficient and less user-spammy to edit the existing embed and change its content to a cancel message